synapse/config/appservice.py

```diff
@@ -47,6 +48,12 @@ def generate_config_section(cls, **kwargs: Any) -> str:
         # enables MAU tracking for application service users.
         #
         #track_appservice_user_ips: true
+
+        # Set to establish maximum backoff (in seconds) between HS -> AS connection attempts.
+        # Upon failing to push appservice events the homeserver will wait
+        # an increasing amount of seconds between retries. This sets an upper limit of that (in seconds)
```

Comment:
Suggested rewording:

```suggestion
        # Set to establish a maximum backoff (in seconds) between HS -> AS connection attempts.
        # Upon failing to push appservice events, the homeserver will reattempt connection to the
        # application service after a delay. The delay increases with subsequent retries.
        # This value sets an upper limit on that delay.
```